packages/node_modules/@webex/internal-plugin-device/src/interceptors/device-url.js

```diff
@@ -45,7 +45,7 @@ export default class DeviceUrlInterceptor extends Interceptor {
 
         // Check if service is not one of the invalid services
         // Assign the url to the device header
-        if (!invalidServices.includes(serviceName)) {
+        if (!invalidServices.includes(serviceName) && serviceName) {
```

Comment:
nitpick alert: you could do `serviceName &&` instead and save upwards of a millisecond of processing time by not looping through the array